site/_guides/fips.md

```diff
@@ -0,0 +1,124 @@
+---
+title: FIPS
+layout: page
+---
+
+The [Federal Information Processing Standard (FIPS) 140-2](https://csrc.nist.gov/publications/detail/fips/140/2/final) publication describes United States government approved security requirements for cryptographic modules.
```

Comment:
TODO: Move the links to the enumerated list at the bottom of the document once we get a pass and determine these links are good